content/docs/examples/bookinfo/index.md

```diff
@@ -66,42 +66,38 @@ To start the application, follow the instructions corresponding to your Istio ru
 If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
 {{< /tip >}}
 
-1. Change directory to the root of the Istio installation directory.
-
-1.  Bring up the application containers:
-
-    *   If you are using [manual sidecar injection](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection),
-        use the following command
-
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo.yaml@)
-        {{< /text >}}
-
-        The `istioctl kube-inject` command is used to manually modify the `bookinfo.yaml`
-        file before creating the deployments as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject).
+1.  Change directory to the root of the Istio installation directory.
 
-    *   If you are using a cluster with
-        [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection)
-        enabled, label the `default` namespace with `istio-injection=enabled`
+1.  The default Istio installation uses [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+    Label the namespace that will host the application with `istio-injection=enabled`:
 
-        {{< text bash >}}
-        $ kubectl label namespace default istio-injection=enabled
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl label namespace default istio-injection=enabled
+    {{< /text >}}
 
-        Then simply deploy the services using `kubectl`
+1.  Deploy your application using `kubectl` as usual:
 
-        {{< text bash >}}
-        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
+    {{< /text >}}
 
-    Either of the above commands launches all four microservices as illustrated in the above diagram.
+    The above command launches all four microservices as illustrated in the above diagram.
     All 3 versions of the reviews service, v1, v2, and v3, are started.
 
     {{< tip >}}
     In a realistic deployment, new versions of a microservice are deployed
     over time instead of deploying all versions simultaneously.
     {{< /tip >}}
 
+    *Note: If you disabled automatic sidecar injection during installation and rely on [manual sidecar injection]
+    (/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection), use the `istioctl kube-inject` command
+     to modify the `bookinfo.yaml` file before deploying your apoplication,
+     as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject):*
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo.yaml@)
+    {{< /text >}}
```

Comment:
Oh, if you want to include the command, then I guess you can't use a tip, as suggested above. Maybe best, is to just make the previous part an ordinary paragraph, i.e., no Note:
```
If you disabled automatic sidecar injection during installation and rely on [manual sidecar injection],
as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject), use the following command
instead:
```

Reply:
See my previous comment. Do not include the command. We cannot provide the necessary context for it to be useful to someone reading this doc. We have an entire document covering manual sidecar injection and we should try to keep this flow simple.

Reply:
Are you both saying this section should be removed?

Reply:
I would keep the tip, but I think you need to delete the `text bash` command, because you can't nest it inside of the tip.

I would suggest changing it to `{{ warning }}`, instead of tip, and move it up just after the command, but before the text "The above command ...".
```
    {{< warning >}}
    If you disabled automatic sidecar injection during installation and rely on [manual sidecar injection]
    (/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection), use the `istioctl kube-inject` command
    to modify the `bookinfo.yaml` file before deploying your apoplication,
    as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject).
    {{< /warning >}}

    The above command launches all four microservices as illustrated in the above diagram.
    All 3 versions of the reviews service, v1, v2, and v3, are started.
```

Reply:
It does not work indeed, I moved it outside of the tip.
This is still a tip, not a warning.

Reply:
I checked and it works, but the linter complains, so it does not really work for now.